packages/tldraw/src/lib/tools/SelectTool/childStates/ScribbleBrushing.ts

```diff
@@ -51,8 +53,15 @@ export class ScribbleBrushing extends StateNode {
 		this.editor.scribbles.stop(this.scribbleId)
 	}
 
+	override onTick = () => {
+		if (this.isDirty) {
+			this.isDirty = false
+			this.updateScribbleSelection(true)
+		}
```

Comment:
I wonder whether we should run these in `onComplete` too? To get the absolute final positions. Probably not gonna be noticeable unless the frame rate gets quite low tbh.